submit-api/src/submit_api/resources/submission.py

```diff
@@ -55,6 +54,10 @@ def post(submission_item_id):
         """Create a submission."""
         create_submission_data = CreateSubmissionRequestSchema().load(API.payload)
         created_submission = SubmissionService.create_submission(submission_item_id, create_submission_data)
+        item_id = create_submission_data.get("item_id")
+        status = create_submission_data.get("status")
+        if status:
+            SubmissionService.update_submission_item_status(item_id, status)
```

Comment:
this kind of check shouldn't be here. and just pass create_submission_date to the method you don't need to pick out each attribute